docs/consume-packages/Package-Source-Mapping.md

```diff
@@ -103,11 +103,13 @@ To learn more about how package installation works, see [the conceptual document
 To fully onboard your repository you may take the following steps:
 
 1. Declare a new [global packages folder for your repo](../reference/nuget-config-file.md#config-section).
-1. Run [`dotnet list package --include-transitive`](/dotnet/core/tools/dotnet-list-package#synopsis) to view all top-level and transitive packages in your solution.
+2. Run [dotnet restore](/dotnet/core/tools/dotnet-restore) to restore dependencies.
+3. Run [`dotnet list package --include-transitive`](/dotnet/core/tools/dotnet-list-package#synopsis) to view all top-level and transitive packages in your solution.

```

Comment:
Thanks for adding these steps. 

Can you please revert the numbering changes?

Reply:
These numbers are added by GitHub when I clicked on `Edit this document` on docs.microsoft.com. I am happy to revert those changes.

Reply:
Reverted number changes in the document. Now the changes proposed in this PR can be noticed easily when compared to earlier. Thanks Nikolche.

Reply:
Using the `1.` numbers allows easier reordering as well, because when you use `1.` it autogens the numbers.